.github/workflows/crossbow-nightly-report-r-tests.yml

```
@@ -0,0 +1,53 @@
+name: Crossbow nightly report R tests
+
+on:
+  push:
+    branches: [ main ]
+    paths:
+      - 'crossbow-nightly-report/**'
+  pull_request:
+    branches: [ main ]
+    paths:
+      - 'crossbow-nightly-report/**'
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: ./crossbow-nightly-report
+
+    steps:
+    - uses: actions/checkout@v3
+
+
+    - name: Install system dependencies
+      run: |
+        sudo apt-get update
+        sudo apt-get install -y libcurl4-openssl-dev libssl-dev libxml2-dev
+
+    - name: Set up R
+      uses: r-lib/actions/setup-r@v2
+      with:
+        r-version: '4.5.0'
```

Same rationale as the other comment. 
```suggestion
        r-version: 'renv'
```